dev_tools.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Thanks for taking the initiative with the shell and batch scrips! After thinking on it, I feel more comfortable if we simply did this work with the Maven/Gradle adoption here soon so that we can just use the platform's specific programs to manage dependencies instead of through one-off scrips.

What do you think? We able to just allow users to manually download Checkstyle for now and we can revisit this when we adopt Maven/Gradle?

Reply:
I have already added the Maven plugin, so yes, we can remove these scripts.